GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -46,6 +46,14 @@ public class SparseVerb : GVFSVerb.ForExistingEnlistment
             HelpText = "List of folders in the sparse set for determining what to project.")]
         public bool List { get; set; }
 
+        [Option(
+            'c',
+            "clean",
+            Required = false,
+            Default = false,
+            HelpText = "Clean up any folders that are not in the list of sparse folders.")]

```

Comment:
Is it clear what `Clean` means in this context? would `Remove folders that are not in the sparse list` make it more obvious that we will remove them?